fluent-bit-to-loki/pkg/buffer/buffer_test.go

```diff
@@ -0,0 +1,139 @@
+package buffer
```

Comment:
Shouldn't it be ?
```suggestion
package buffer_test
```

Reply:
Sorry but I am using not exported function from the buffer package.
I don't see other way to test it.

Reply:
Ok, I got it. Thanks!